content/Products/OpenshiftMonitoring/telemetry.md

```diff
@@ -54,15 +54,18 @@ If you are above these limits, you have 2 choices:
 
 Finally your metric **MUST NOT** contain any personally identifiable information (names, email addresses, information about user workloads).
 
-Use the following template to file a JIRA task in the [MON project](https://issues.redhat.com/projects/MON).
+Use the following information to file 1 JIRA ticket per metric in the [MON project](https://issues.redhat.com/projects/MON):
```

Comment:
Howe about we populate the link? Something like 
```suggestion
Use the following information to file 1 JIRA ticket per metric in the [MON project](https://issues.redhat.com//secure/CreateIssueDetails!init.jspa?pid=12323177&issueType=3&labels=telemetry-review-request&summary=Telemetry+request+PROJECT&description=h1.%20Request%20for%20sending%20data%20via%20telemetry%0A%0AThe%20goal%20is%20to%20collect%20metrics%20about%20...%20because%20...%0A%0Ah2.%20%3CMetric%20name%3E%0A%0A%3CMetric%20name%3E%20represents%20...%0A%0ALabels%0A%2A%20%3Clabel%201%3E%2C%20possible%20values%20are%20...%0A%2A%20%3Clabel%202%3E%2C%20possible%20values%20are%20...%0A%0AThe%20cardinality%20of%20the%20metric%20is%20at%20most%20%3CX%3E.%0A%0AComponent%20exposing%20the%20metric%3A%20https%3A%2F%2Fgithub.com%2F%3Corg%3E%2F%3Cproject%3E%0Ah2.%20%3COther%20metric%3E%0A%0A...):
```

Reply:
done